QUEST.tsv

```diff
@@ -2019,9 +2019,9 @@ QUEST_20150511_002022	$When a caster makes a curse, the caster has a tremendous
 QUEST_20150511_002023	$The Complete Research Materials of Varkis
 QUEST_20150511_002024	$The First Research Material{nl}-{nl}We've finally arrived in front of the Zachariel's Mausoleum. {nl}He was the first king of this kingdom and used an enormous number of people to build it.{nl}I believe only this place can answer the question I've had in my mind for a long time.
 QUEST_20150511_002025	$What was he trying to protect with this gigantic tomb? {nl}The answer must be inside, but the bleak atmosphere the inside exudes makes me hesitate to go in.
-QUEST_20150511_002026	$The Second Research Material{nl}-{nl}There's something inside the dark tomb.{nl}As I advanced inside, the large stone statues all turned to look down at me and then chased me out. {nl}I guess the great king in the past is still the great king in the present. {nl}I better investigate from the pole outside of the tomb.
+QUEST_20150511_002026	$The Second Research Material{nl}-{nl}There's something inside the dark tomb.{nl}As I advanced inside, the large stone statues all turned to look down at me and then chased me out. {nl}I guess the great king in the past is still the great king in the present. {nl}I better investigate from the pillar outside of the tomb.
 QUEST_20150511_002027	$The Third Research Material{nl}-{nl}With the help from the young, kind historian, I was able to find a big clue. {nl}Yes. The best place to hide a tree is in a forest. {nl}Ruko Plateau and the Plateau of Rex… I'm so close I can almost touch it. {nl}I'm confident the Goddess sent him to me. May the blessing of the Goddess always be with us!
```

Comment:
With the help from the young, kind historian, -> With the help of a young, kind historian,?
Ruko Plateau -> Rukas Plateau